src/main/c/seasocks/Connection.h

```diff
@@ -30,6 +30,7 @@
 #include "seasocks/ResponseWriter.h"
 #include "seasocks/TransferEncoding.h"
 #include "seasocks/ZlibContext.h"
+#include "internal/HybiPacketDecoder.h"
```

Comment:
A possible solution to avoid this could be to move the decoder part into a separate internal class. Using a forward declaration doesn't expose the internal API.